crypto/evp/evp_enc.c

```diff
@@ -471,6 +543,62 @@ int EVP_CipherUpdate(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl,
         return EVP_DecryptUpdate(ctx, out, outl, in, inl);
 }
 
+int EVP_CipherPipelineUpdate(EVP_CIPHER_CTX *ctx,
+                             unsigned char **out, int *outl,
+                             const unsigned char **in, int *inl)
```

Comment:
Should we make these lengths `size_t`?

Reply:
I think the same. The only reason I went with `int` is to match the `EVP_CipherUpdate()` and `EVP_CIpherFinal()` variants. Is it okay to break uniformity?

Reply:
IMO, yes.

Reply:
Yes for me as well.